lib/live_debugger_refactor/services/callback_tracer/gen_servers/tracing_manager.ex

```diff
@@ -29,32 +31,32 @@ defmodule LiveDebuggerRefactor.Services.CallbackTracer.GenServers.TracingManager
   def handle_info(:setup_tracing, state) do
     TracingActions.setup_tracing!()
 
-    {:noreply, state}
+    noreply(state)
```

Comment:
Imo when you don't pipe you should use standard convention `{:noreply, state}`. These are only helpers to make piping looks nicer